_tools/epfl_infrastructure.md

```diff
@@ -0,0 +1,159 @@
+
+You need to do this every few months, as offline accesses have an expiration date for security; you'll receive an email reminding you of this when the date arrives.
+
+Camipro machines also let you insert your card and rewrite the expiry date; this does not update your rights, and is only useful to present your card outside of EPFL to get student-related discounts.
+
+Check your Camipro rights on [https://mycamipro.epfl.ch/client/seracc](https://mycamipro.epfl.ch/client/seracc).
+
+
+## Trash
+
+Need to dispose of something bigger or weirder than what the usual recycling points can handle?
+Try the IC trash room, [INM 093.5](https://plan.epfl.ch//?room==INM%20093.5).
+In addition, next to it is a "help yourself" point where you can leave stuff that's still usable but you just don't want any more, and take whatever others have left.
+There is also a smaller trash room for IC in [BC 902](https://plan.epfl.ch/?room==BC%20902).
+
+If that's not enough, EPFL's main trash collection point is below Esplanade, where you can recycle all kinds of things including wood.
+
+Note that "paper" and "cardboard" are two different kinds of trash, and cardboard must always, always, always be folded flat before being put into a container.
+
+Lyreco cardboard boxes should be returned intact to the mail room, not trashed.
+
+
+## Webcam and headset
+
+Still have an old phone? Exchange it for a webcam and headset to use with Jabber [here](https://support.epfl.ch/epfl?id=epfl_sc_cat_item&sys_id=733e40414f3c43809d2bdf601310c73f).
+
+
+## Markdown in Moodle
+
+Moodle's default WYSIWYG editor isn't everyone's cup of tea.
+
+Go to your Moodle profile > Editor preferences and select "Plain text area".
+
+Then, when you write a message, pick Markdown in the format options.
+
+
+## LDAP
+
+Need more information than people.epfl.ch can provide? Want to search for details? Convert a SCIPER to a GASPAR?
+Use the `ldapsearch` command-line tool on [ldap.epfl.ch](ldap.epfl.ch), e.g., `ldapsearch -h ldap.epfl.ch -b 'o=epfl,c=ch' -LLL -x uniqueIdentifier=SCIPER`.
+
+
+## File storage
+
+All IC personnel have a personal directory on two network file systems. Both support NFS and CIFS/SMB, although the latter is infinitely easier to use in practice.
+
+**MyNAS** is general-purpose space. All EPFL personnel have 25 GB by default. You can mount your remote directory by installing the `cifs-utils` and `keyutils` packages then running
+`sudo mount.cifs //files<X>.epfl.ch/data/<gaspar> -o user=<gaspar>,uid=<local-UID>,gid=<local-GID>,dir_mode=0700,dynperm,vers=3.1.1,sec=ntlm  /local/mount/point`
+(replace `<X>` with the server number indicated at [mynas.epfl.ch](https://mynas.epfl.ch), `<gaspar>` with your GASPAR username,
+and `<local-UID>` & `<local-GID>` with the UID and GID of the user on your machine who should have access to the mounted directory;
+you can change `dir_mode` and additionally `file_mode` for finer control of local permissions)
```

Comment:
MyNAS also supports WebDAV which is good because it lets you use it for Zotero file storage (which is only 300 MB by default on free accounts).

Reply:
done